### DISCUSSION TITLE
Multi-apns and VLANs in BRIDGE MODE

### DISCUSSION THREAD

**Discussion by pkapatsos-blinq** [magma/magma#5421] [Q&A]
Hi when using bridge-mode or setting for MULTI-APNS via DHCP and vlans
What is the AGW interface eth0 setup for this mode tagged or untagged ?

**Comment by pshelar** [comment]
@pkapatsos-blinq In Bridged mode eth0 is part of uplink_br0. so we can not directly configure eth0, all configuration is done on uplink_br0 interface.
Are you looking for static or DHCP based config for SGi interface?

**Reply by pkapatsos-blinq** [reply]
@pshelar Ok am then looking to Uplink_br0 , does it send tagged or untagged packets? as need both options for SGi. 
Also please Where or what file has the uplink_br0 ip static setting ? How can I reset back via files? 
As these changes when in bridge mode or these, 
![image](https://user-images.githubusercontent.com/61711855/110819198-f0e36000-825b-11eb-81a7-45efe621bd9a.png)

can render AGW unusable  ,similar to this issue https://github.com/magma/magma/issues/4322 .. services do not stop and NIC  has no gateway ip route set looses Orc8 or SGi side NIC . I cant determine if its our network setup or bugs.
I just want to get this back without rebuild. as the immediate can you dm via slack  please.

A recap of the issue , we have a Cisco L3 switch and unlike mikrotic you cant have tagged and untagged on the same port when its a trunk.  I confirmed that freedomfi have this working only with 2 upstream routers/switches types a Mikrotic/Ubiquity , NOT a Cisco L3 . 
You saw our issues already on getting DHCP and Mullit-vlan IP to work and only with your fix  via https://github.com/magma/magma/pull/4338 did this work with an access port. with custom L3 settings for 2nd APN.

We just want to highlight or share that a number of settings when making them in bridge mode fail and cant confirm if its our environment or the Orc8r / AGW releases we  are using as do not have another type to test.
However all work when in NAT mode.
 
I have tired on several AGW, all have unpredictable results when going to bridge mode first when setting these SGI mgmt ip or SGi default gateway and or vlan id , however as stated is in you are NAT mode set the SGi mgmt IP or Gateway . then go to bridge mode not issues setting all work.

Any chance you can share  what is your router/Switch device for your bare-metal testing and are you testing your settings in Bridge-mode  or NAT for verifications ?
